### PR TITLE
Adds CGP-158 : Celo Infrastructure for Chain Lifecycle Operational Support (CICLOPS)

### PR DESCRIPTION
**Pull request by ericnakagawa**
This proposal corresponds with the forum post: https://forum.celo.org/t/celo-infrastructure-for-chain-lifecycle-operational-support-ciclops/9661

### CODE REVIEW

**Review by work-0xj4an — APPROVED** [review]

File: CGPs/cgp-0158/mainnet.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[
+    {
+      "contract": "GoldToken",
+      "address": "0x471EcE3750Da237f93B8E339c536989b8978a438",
+      "function": "Approve",
+      "args": [
+        "0xd71efa410B6EaAB0bAc3b515B393C886BE70F09E",
+        "3838900000000000"
```
Comment (0xzoz) [inline CGPs/cgp-0158/mainnet.json:8]: The amounts are represented in gwei when the protocol expects them in wei.

Where `3838900000000000` gwei should be `3838900000000000000000000` wei

File: CGPs/cgp-0158/mainnet.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      ],
+      "value": "0"
+    },
+    {
+      "contract": "StableToken",
+      "address": "0x765DE816845861e75A25fCA122bb6898B8B1282a", 
+      "function": "Approve",
+      "args": [
+        "0xd71efa410B6EaAB0bAc3b515B393C886BE70F09E",
+        "3000000000000000"
```
Comment (0xzoz) [inline CGPs/cgp-0158/mainnet.json:18]: The amounts are represented in gwei when the protocol expects them in wei.

Where `3000000000000000` gwei should be `3000000000000000000000000` wei

**Review by 0xzoz — CHANGES_REQUESTED** [review]

File: CGPs/cgp-0158/mainnet.json (hunk 1 above)
Comment (ericnakagawa) [reply CGPs/cgp-0158/mainnet.json]: fixed

File: CGPs/cgp-0158/mainnet.json (hunk 2 above)
Comment (ericnakagawa) [reply CGPs/cgp-0158/mainnet.json]: fixed

**Review by 0xzoz — APPROVED** [review]

**Pull request merged**